src/components/PopularFrequentSearchCategories/PopularFrequentSearchCategories.tsx

```diff
@@ -11,32 +10,34 @@ const PopularFrequentSearchCategories = ({
   PopularFrequentSearchTitle,
   onSelectPopularItem,
 }: PopularFrequentSearchProps) => {
-  const { spacing, palette, typography } = useTheme();
-
+  const theme = useTheme();
+  const mobile = useMediaQuery(theme.breakpoints.between("xs", "sm"));
```

Comment:
The outcome does not match the [Figma Design](https://www.figma.com/file/JexGuMEfOIWqn6KX8SBy5C/Forkfacts-with-material-theme-(Copy-Colours)?node-id=3155%3A114494&t=JRVKXPqXaYJNtTLd-4)

Reply:
fixed